src/fivetran-destination/src/destination/dml.rs

```diff
@@ -50,6 +50,29 @@ pub async fn handle_truncate_table(request: TruncateRequest) -> Result<(), OpErr
         SystemTime::UNIX_EPOCH + Duration::new(secs, nanos)
     };
 
+    let (_dbname, client) = config::connect(request.configuration).await?;
+
+    let exists_stmt = format!(
+        r#"
+        SELECT EXISTS(
+            SELECT 1 FROM mz_tables t
+            LEFT JOIN mz_schemas s
+            ON t.schema_id = s.id
+            WHERE s.name = $1 AND t.name = $2
+        )"#
+    );
+    let exists: bool = client
+        .query_one(&exists_stmt, &[&request.schema_name, &request.table_name])
+        .await
+        .map(|row| row.get(0))
+        .context("checking existence")?;
```

Comment:
I might be missing something, but how does the `1` become a bool?

Reply:
That's what the `EXISTS` part of the statement does. If any rows are returned then `EXISTS` resolves to `true`